src/components/DocumentViewer.tsx

```diff
@@ -1,22 +1,36 @@
-// @ts-ignore
-import React, { useRef, useEffect } from 'react'
-import useInstances from '../context'
+import React from 'react';
+import useInstance from '../context';
+
+const disabledElements = [
+  'header',
+  'toolsHeader',
+  'viewControlsButton',
+  'leftPanelButton',
+  'pageNavOverlay',
+  'searchButton',
+  'menuButton',
+  'textPopup',
+  'contextMenuPopup',
+];
 
 export type TProps = {
-  docUrl: string
-  UID: string
+  zoom: number
+  // will add more initial configs
 }
 
-function DocumentViewer({ docUrl, UID }: TProps): JSX.Element {
-  const viewer = useRef<HTMLDivElement>(null)
+const DocumentViewer = React.forwardRef<HTMLDivElement, TProps>((props, ref) => {
```

Comment:
This component is supposed to create the instance of WebViewer - please make sure you understand the spec before making changes

Reply:
It now creates user supplied WV instance. 
For the forwardRef from users, Typescript gives me the error ` Property 'current' does not exist on type '(instance: TRef | null) => void' if I don't add `ref as React.RefObject<HTMLDivElement>`. With my limited knowledge, I couldn't find a way to better type it even after a search. @lbittner-pdftron